Source/Tests/FEXLoader.cpp

```diff
@@ -223,7 +233,7 @@ int main(int argc, char **argv, char **const envp) {
   }
 
   // Ensure RootFS is setup before config options try to pull CONFIG_ROOTFS
-  if (!FEX::RootFS::Setup(envp)) {
+  if (!FEXServerClient::SetupClient(argv[0])) {
     LogMan::Msg::EFmt("RootFS failure");
```

Comment:
Shouldn't this be updated from "RootFS ..."?

Reply:
Resolved.